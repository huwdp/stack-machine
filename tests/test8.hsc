pushf 1.2
pushf 2.9
addf
printl
pushf 0.25
subf
printl
pushf 1.2
mulf
printl
pushf 3.4
divf
printl