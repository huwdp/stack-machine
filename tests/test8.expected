Welcome to Huw's Stack Machine
4.1000004
3.8500004
4.620001
1.3588238
